assignment/scripts/loops-practice.js

```diff
@@ -11,37 +11,71 @@ for (let i=0; i<4; i++) {
 }
 
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
-//   - Which part of the example loop do you need to change to do this?
+//   - Which part of the example loop do you need to change to do this? 
+// changed the number "4" to "6"
 console.log('count from 0 to 5');
 
+for (let i=0; i<6; i++) {  
```

Comment:
FWIW, it is more common to spaces around the operators here (other than `++`).

```suggestion
for (let i = 0; i < 6; i++) {  
```